CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+
+## 17.13.0
+
+* Add support for description text
```

Comment:
The other entries have a like to the PR. Is that the standard for this repo?

Reply:
Yes sorry leena - added